src/z3c/schema/payments/field.py

```diff
@@ -11,7 +11,7 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""ISO 7812 Credit Card
+"""
```

Comment:
I'm sure you had a good reason to remove module docstrings.  I'm curious to know it.